api/librdb-api.h

```diff
@@ -202,14 +203,15 @@ typedef struct RdbHandlersDataCallbacks {
     RdbRes (*handleStringValue)(RdbParser *p, void *userData, RdbBulk str);
     RdbRes (*handleListElement)(RdbParser *p, void *userData, RdbBulk str);
 
-    /*** TODO: RdbHandlersDataCallbacks: handleHashElement, handleSetElement, handleZsetElement, handleStreamElement, handleStreamMetadata, handleStreamCgroup  ***/
+    /*** TODO: RdbHandlersDataCallbacks: handleHashElement, handleSetElement, handleZsetElement ***/
     RdbRes (*handleHashElement)(RdbParser *p, void *userData, RdbBulk field, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleSetElement)(RdbParser *p, void *userData, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleZsetElement)(RdbParser *p, void *userData, RdbBulk elm, double score, uint64_t totalNumElm);
-    RdbRes (*handleStreamElement)(RdbParser *p, void *userData, streamID *id, RdbBulk field, RdbBulk value, uint64_t totalNumElm);
-    RdbRes (*handleStreamMetadata)(RdbParser *p, void *userData, uint64_t len, uint64_t entriesAdded, streamID *firstID, streamID *lastID, streamID *maxDeletedEntryID);
-    RdbRes (*handleStreamCgroup)(RdbParser *p, void *userData, streamCG *cgroup); /* when succeeds, 'cgroup' belongs to the calee  */
-
+    /*** TODO: RdbHandlersDataCallbacks: handleStreamNewGroup, handleStreamMetadata, handleStreamElement, handleStreamConsumerPendingEntry ***/
+    RdbRes (*handleStreamNewGroup)(RdbParser *p, void *userData, RdbBulk grpName, RdbStreamID lastId);
```

Comment:
don't we need a CB for new consumer as well?
and maybe per group and per consumer metadata CB?